src/openms/include/OpenMS/KERNEL/MobilityPeak1D.h

```diff
@@ -128,6 +128,18 @@ namespace OpenMS
       position_ = position;
     }
 
+    /// Alias for getIM()
+    inline CoordinateType getMZ() const
```

Comment:
I think we should not use getMZ as an alias for getMZ but instead change the algorithm to use
getPos()